packages/exhale/build.sh

```diff
@@ -6,4 +6,4 @@ TERMUX_PKG_MAINTAINER="@DLC01"
 TERMUX_PKG_VERSION=1.1.6
 TERMUX_PKG_SRCURL=https://gitlab.com/ecodis/exhale/-/archive/v${TERMUX_PKG_VERSION}/exhale-v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=bf817c42866294ce1c3187d5b09d9e48e46d36ef2ba8ff3b7630e87c5113ea9d
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DBUILD_SHARED_LIBS=ON, -DCMAKE_BUILD_TYPE=Release"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Release"
```

Comment:
Btw,  why are you adding these flags? I see no mention of the first one in that repo and [the second is the default configuration anyway](https://gitlab.com/ecodis/exhale/-/blob/master/CMakeLists.txt#L22).

Reply:
Looks like you were right to add the first flag, as it is now adding a static library, which we don't prefer. Please add the first flag back and we will get this in.